tests/plugins/toolsets/kubernetes/test_kubernetes_logs.py

```
@@ -0,0 +1,238 @@
+        params = FetchLogsParams(
+            namespace="default",
+            pod_name="test-pod",
+            limit=2000,
+            start_time=None,
+            end_time=None,
+            filter_pattern=None,
+        )
+
+        result = self.toolset.fetch_logs(params=params)
+
+        self.assertEqual(result.status, ToolResultStatus.SUCCESS)
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)